packages/falso/src/lib/utils/objectKeys.ts

```diff
@@ -0,0 +1,17 @@
+export type ObjectKeys<T extends Record<PropertyKey, unknown>> = Exclude<
+  keyof T,
+  symbol
+>;
+
+/**
+ * A utility function which behaves identical to
+ * [Object.keys()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys).
+ * It preserves type strictly
+ */
+const objectKeys = <Type extends Record<PropertyKey, unknown>>(
```

Comment:
please use named export

Reply:
I have completed the modification as you suggested!
I don't think there will be a situation to return other functions in that file(objectKeys.ts). Is there an advantage when use export named?

Reply:
Oh I learned from you so many things
thank you😃